netci.groovy

```diff
@@ -14,17 +14,10 @@ def windowsUnitTestMachine = 'win2016-base'
 
 static void addRoslynJob(def myJob, String jobName, String branchName, Boolean isPr, String triggerPhraseExtra, Boolean triggerPhraseOnly = false) {
   def archiveSettings = new ArchivalSettings()
-  archiveSettings.addFiles('Binaries/**/*.pdb')
-  archiveSettings.addFiles('Binaries/**/*.xml')
-  archiveSettings.addFiles('Binaries/**/*.log')
-  archiveSettings.addFiles('Binaries/**/*.dmp')
-  archiveSettings.addFiles('Binaries/**/*.zip')
+  archiveSettings.addFiles('Binaries/Debug/Logs/**)
+  archiveSettings.addFiles('Binaries/Release/Logs/**)
```

Comment:
Why not `Binaries/*/Logs/**`?

Reply:
There are other directories in Binaries and I don't want to accidentally grab their Logs directory here.